### DISCUSSION TITLE
Printer Users Presets Missing

### DISCUSSION THREAD

**Discussion by slammers67** [OrcaSlicer/OrcaSlicer#7257] [General]
I am using Orca 2.2.0-RC portable and loading it off a netwrok share (windows environment, Windows 10)

I added a new custom printer and it shows up under the User Presets.

However, it seems that the user presets only show up every second time I Open orca.

Load Orca -> user presets missing
Close Orca and Reopen -> User Presets Visible
Close Orca and Reopen -> User presets missing

Has anyone else seen this? Is this some sort of locked file issue?

Also, after I setup my customer printer on one PC, I copied the Orcaslicer directory contents from my AppData folder to the Orca Portable network share into data_dir. I expected my user presets to then show up when loading from a different PC. Although my system presets seemed to be shared from the data_dir, again the User Presets are missing. I did verify the the User on the home page is selected correctly.

**Comment by urself25** [comment]
As stated in [this conversation](https://github.com/SoftFever/OrcaSlicer/discussions/1594)

Default presets are in C:\Users%USERNAME%\AppData\Roaming\OrcaSlicer\system as .json files
User presets are in C:\Users%USERNAME%\AppData\Roaming\OrcaSlicer\user\ (some number) \ as .json files

**Reply by slammers67** [reply]
I know where the presets are. My problem is that every second time I open Orca slicer, they are missing from the user interface.

This same problem is happening on the reloeasewd version of 2.2.0

**Reply by urself25** [reply]
Then you'll need to use the orcaslicer_profile_validator to find what's the issue.

**Comment by arunshivaram** [comment]
I am observing the same issue that the user preset printer configs load only if Orca is closed and re-opened. 

I'm also using v2.2.0 but on a mac if that is helpful

**Comment by Boogabaga** [comment]
I am also experiencing this same issue. Running v2.2.0 on a Win10 PC (also did it on the previous version, thought updating might fix it, but

it didn't). It's only slightly annoying to have to do this, but what's more annoying is not knowing _why_ it is doing it. Unfortunately, I have no

real value to add towards resolving this issue, other than saying, "Yep, it's real, and quite peculiar."

**Comment by woody4165** [comment]
I'm experiencing same problem with system filament profiles (I'm on Mac and I know where system and user folders are).
And I'm also not able to create new custom filament (I have opened a new thread in Q&A section)

**Comment by Chreutz** [comment]
I might have found a solution. I used an existing MK4S profile as a template to create a profile for my MK3-based klipper printer. That caused the profiles to 'disappear' from my user defined printer after a restart, but they are instead still on the MK4S profile. 

Process profiles have section in their json files that are not exposed in the UI:

```
    "compatible_printers": [],
    "compatible_printers_condition": "printer_notes=~/.*MK4S.*/ and nozzle_diameter[0]==0.4 and printer_notes!~/.*HF_NOZZLE.*/",
```

I needed to add my printer's name into the printer profile notes, and also make sure these conditions match. Either by making the regex patterns match, or by pasting the name directly in the list of compatible printers.

See also: https://www.reddit.com/r/3Dprinting/comments/1c5hcfu/orcaslicer_share_filaments_between_printers/
